logstash-core-event-java/src/main/java/org/logstash/Timestamp.java

```diff
@@ -3,14 +3,14 @@
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
-import org.joda.time.LocalDateTime;
 import org.joda.time.Duration;
+import org.joda.time.LocalDateTime;
 import org.joda.time.format.DateTimeFormatter;
 import org.joda.time.format.ISODateTimeFormat;
 
 import java.util.Date;
 
-@JsonSerialize(using = TimestampSerializer.class)
+@JsonSerialize(using = org.logstash.json.TimestampSerializer.class)
```

Comment:
Wait there is two TimestampSerializer classes, 1) in the `org.logstash` and 2) in `org.logstash.json`!!

@talevy @colinsurprenant - Which one do we keep?